tests/saitests/sai_qos_tests.py

```diff
@@ -2140,9 +2146,22 @@ def runTest(self):
                     pkts_num -= diff
                     expected_wm -= diff * cell_occupancy
                     fragment = total_shared - expected_wm
+
+                pkts_total += pkts_num
```

Comment:
Move this line below the branch of cisco-8000 as it is just used here.

Reply:
Moved occurrences of pkts_total to within cisco contexts.